### ISSUE TITLE
issues when spacing in directory

### ISSUE THREAD

**Issue by gre** [mattdesl/budo#14]
I have a filesystem with a space in my path: `/Volume/Case Sensitive/...`

And when I try to use `--outfile`:

```
{ [Error: EISDIR, open '/Volumes/Case'] errno: -21, code: 'EISDIR', path: '/Volumes/Case' }
```

Regards,
gre

**Comment by mattdesl** [comment]
Good catch :+1: 

Should be fixed in 1.3.2, could you test?

**Comment by mattdesl** [comment]
should be fixed in 3.0.0

Spaces are escaped for the server entry point, using `url` module.
